DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/matmul/README.md

```diff
@@ -0,0 +1,271 @@
+
+### Compiler Flags Used
+
+| Flag                  | Description
+|:---                   |:---
+| `-Xshardware`         | Target FPGA hardware (as opposed to FPGA emulator)
+| `-Xsclock=360MHz`     | The FPGA backend attempts to achieve 360 MHz
+| `-Xsseed`             | Specifies the Intel® Quartus® compile seed, to yield slightly higher fmax
+
+Additionaly, the cmake build system can be configured using the following parameters:
+
+| cmake option              | Description
+|:---                       |:---
+| `-DDEBUG` | Used to turn on/off debug information (printing out matrices)
+| `-DROWS_A` | Specifies the number of rows of matrix A
+| `-DCOMMON` | Specifies the number of columns of matrix A or rows of matrix B (must be equal)
+| `-DCOLS_B` | Specifies the number of columns of matrix B
+| `-DTILE_A` | Specifies the tile size used on matrix A
+| `-DTILE_B` | Specifies the tile size used on matrix B
+
+## Build the `Matrix multiply` Design
+
+> **Note**: When working with the command-line interface (CLI), you should configure the oneAPI toolkits using environment variables. 
+> Set up your CLI environment by sourcing the `setvars` script located in the root of your oneAPI installation every time you open a new terminal window. 
+> This practice ensures that your compiler, libraries, and tools are ready for development.
+>
+> Linux*:
+> - For system wide installations: `. /opt/intel/oneapi/setvars.sh`
+> - For private installations: ` . ~/intel/oneapi/setvars.sh`
+> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
+>
+> Windows*:
+> - `C:\Program Files(x86)\Intel\oneAPI\setvars.bat`
+> - Windows PowerShell*, use the following command: `cmd.exe "/K" '"C:\Program Files (x86)\Intel\oneAPI\setvars.bat" && powershell'`
+>
+> For more information on configuring environment variables, see [Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html) or [Use the setvars Script with Windows*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+### On Linux*
+
+1. Change to the sample directory.
+2. Configure the build system for **Intel® PAC with Intel Arria® 10 GX FPGA**, which is the default.
+
+   ```
+   mkdir build
+   cd build
+   cmake ..
+   ```
+   For **Intel® FPGA PAC D5005 (with Intel Stratix® 10 SX)**, enter the following:
+   ```
+   cmake .. -DFPGA_BOARD=intel_s10sx_pac:pac_s10
+   ```
+3. Compile the design. (The provided targets match the recommended development flow.)
+
+   1. Compile for emulation (fast compile time, targets emulated FPGA device).
+      ```
+      make fpga_emu
+      ```
```

Comment:
Please add a simulation target

Reply:
done. Should I base this off another code sample's readme that supports simulation? I couldn't seem to find one.

Reply:
If you look at the development branch in the code samples repo, 99% of the samples feature simulation now